lib/src/qr_code.dart

```diff
@@ -19,6 +19,8 @@ class QrCode {
   List<int> _dataCache;
   final List<QrByte> _dataList = <QrByte>[];
 
+  List<List<bool>> get modules => _modules;
```

Comment:
Does this *need* to be public for any use case other than testing?

If yes, just make the final field public...

Reply:
added `@visibleForTesting` annotation